lnbits/helpers.py

```diff
@@ -223,3 +223,15 @@ def decrypt_internal_message(m: Optional[str] = None) -> Optional[str]:
     if not m:
         return None
     return AESCipher(key=settings.auth_secret_key).decrypt(m)
+
+
+def filter_dict_keys(data: dict, filter_fields: Optional[list[str]]) -> dict:
+    if not filter_fields or len(filter_fields) == 0:
+        return {**data}
```

Comment:
```suggestion
        return data
```
you can just return the dict

Reply:
Not the same thing. `{**data}` returns a shallow clone of the original dict.
If there are filters then a shallow clone with filtered fields will be returned.
Therefore `{**data}` handles the `output` consistent no matter the `inputs`.

Reply:
maybe add a comment so someone does not "fix" this in the future?

Reply:
done!